dex-ws-load/build.sbt

```diff
@@ -0,0 +1,17 @@
+enablePlugins(GatlingPlugin)
+
+scalaVersion := "2.12.10"
+
+scalacOptions := Seq(
+  "-encoding", "UTF-8", "-target:jvm-1.8", "-deprecation",
+  "-feature", "-unchecked", "-language:implicitConversions", "-language:postfixOps")
+
+libraryDependencies += "io.gatling.highcharts" % "gatling-charts-highcharts" % "3.3.1" % "test,it"
+libraryDependencies += "io.gatling"            % "gatling-test-framework"    % "3.3.1" % "test,it"
+libraryDependencies += "org.scalactic" %% "scalactic" % "3.0.8"
+libraryDependencies += "org.scalaj" % "scalaj-http_2.11" % "2.3.0"
```

Comment:
It looks like there are 2.4.2 that supports not only Scala 2.11 https://search.maven.org/search?q=scalaj-http
Consider:
```
"org.scalaj" %% "scalaj-http" % "2.4.2"
```